python/tests/scripts/f1_imports.metta

```diff
@@ -1,13 +1,19 @@
 ; NOTE: this behavior can change in the future
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-; Even at the very beginning of the main script `(get-atoms &self)`
-; returns one atom, which wraps the space of stdlib.
+; At the very beginning of the main script, the space
+; contains an atom wrapping the space of stdlib.
 ; The type of this atom is the same as of `&self`
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; QUESTION: Are space implementations really required to preserve ordering of atoms?
```

Comment:
We cannot impose the requirement of preserving the atoms order to all spaces. It will definitely be not the case for many spaces like DAS or potentially neural or hypervector spaces. It is convenient to have the order preservation feature for the in-RAM program space, and some people already rely on this (although we warn them that this may change). It's ok to break order preservation (and it might be less confusing to do this asap), but in this case, we need to provide means to do what these people do with `superpose` and `collapse` (e.g., sequential execution of side-effect functions, sequential traversal of MeTTa expressions, etc.).

Reply:
I looked at this question in the context and would like to add that this question is irrelevant to this situation. The script is parsed and evaluated sequentially. The comment means that there is already `&stdlib` atom placed in `&self` space already at the beginning of the script evaluation. It doesn't mean and doesn't matter that `&stdlib` atom is placed at the beginning of the space. Basically, how this comment is changed in this PR sounds more confusing to me than it was before.  Maybe, the comment requires further clarification and the question can be removed as irrelevant.